Missing library in /external/lua/src

After a rebase tonight, the compile of master failed with the following error:

==========

-- Configuring done
CMake Error at src/external/lua/CMakeLists.txt:36 (add_library):
  Cannot find source file:

    src/lbitlib.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/external/lua/CMakeLists.txt:36 (add_library):
  No SOURCES given to target: lua


CMake Generate step failed.  Build files cannot be regenerated correctly.
=========

I checked and indeed lbitlib.c is missing from the source directory.

OS: Ubuntu 21.04.
Toolchain: llvm-12

--- a/src/external/lua/CMakeLists.txt
+++ b/src/external/lua/CMakeLists.txt
@@ -21,7 +21,6 @@ set(LUA_SOURCES
 	"src/lzio.c"
 	"src/lauxlib.c"
 	"src/lbaselib.c"
-	"src/lbitlib.c"
 	"src/lcorolib.c"
 	"src/ldblib.c"
 	"src/liolib.c"

